extensions/wikia/MonetizationModule/MonetizationModuleHelper.class.php

```diff
@@ -152,22 +146,21 @@ public function getMonetizationUnits( $params ) {
 			];
 			$log->debug( "MonetizationModule: cannot get monetization units.", $loggingParams );
 		} else if ( !empty( $result ) ) {
-			$result = $this->processData( $result, $params );
+			$result = $this->processData( $title, $result, $params );
 		}
 
-		wfProfileOut( __METHOD__ );
-
 		return $result;
 	}
 
 	/**
 	 * Process data for the ad units (include logic to handle the data)
+	 * @param Title $title
 	 * @param string $data - data from API (json format)
 	 * @param array $params - API parameters
 	 * @param boolean $setMemc - set to true to set to memcache
 	 * @return mixed
 	 */
-	public function processData( $data, $params, $setMemc = true ) {
+	public function processData( $title, $data, $params, $setMemc = true ) {
```

Comment:
If the PHPDocs say `$title` is an instance of `Title` let's make the method signature to say it too: `processData( Title $title, $data, $params, $setMemc )` :japanese_ogre: